lib/src/grid/types.ts

```diff
@@ -23,23 +23,92 @@ type PlaceGeneric<PlaceValues, Element extends string> =
   | PlaceValues;
 
 export type GridItemProps = {
+  /**
+   * Sets the name of an item so that it can be referenced by a template created with the grid-template-areas property.
+   */
```

Comment:
I thought about this, the problem is that they may get confused since it doesn't work the same as its CSS counterpart. That's why it has a longer explanation and no link to MDN. What do you think?

Reply:
I understand, I agree!